### PR TITLE
feat: [CDS-86002]: added pdc support

### PR DESCRIPTION
**Pull request by vitsafronovici**
## Describe your changes
added pdc connector support

## Comment Triggers

<details>
  <summary>PR Check triggers</summary>
  
- Build: `trigger build`
- Sub Category Field Check: `trigger subcategoryfieldcheck`
- gitleaks: `trigger gitleaks`

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -20,15 +20,10 @@ require (
 
 require (
 	github.com/Azure/go-ansiterm v0.0.0-20210617225240-d185dfc1b5a1 // indirect
-	github.com/Masterminds/goutils v1.1.1 // indirect
```
Comment (rathodmeetsatish) [inline go.mod:23]: revert these changes and only keep go-sdk version change in this file and correspoding changes in go.sum

**Review by rathodmeetsatish — CHANGES_REQUESTED** [review]

**Comment by bot2-harness** [comment]
This pull request has been automatically closed due to 30 days of inactivity. If you believe this was done in error, please feel free to reopen the PR or reach out to the BT team for further assistance. Thank you for your contributions!
